src/engine-actions/get-total-commit-fees.js

```diff
@@ -0,0 +1,26 @@
+const { Big } = require('../utils')
+const { listChannels } = require('../lnd-actions')
+
+/**
+ * Get total commit fees for all channels for a specific daemon
+ *
+ * @returns {string} totalCommitFees (int64)
+ */
+async function getTotalCommitFees () {
+  const { channels = [] } = await listChannels({ client: this.client })
+
+  if (channels.length === 0) {
```

Comment:
This seems like an unnecessary step since you're doing a reduce - won't it just return `Big(0).toString()` even if there are no channels returned?